examples/huggingface-transformers/README.md

```diff
@@ -0,0 +1,137 @@
+performance with DeepSparse.  For a full list of options run `python 
+benchmark.py -h`.
+
+To run a benchmark using the DeepSparse Engine with a pruned BERT model that uses all available CPU cores and batch size 1, run:
+```bash
+python benchmark.py \
+    zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-moderate \
+    --batch-size 1
+```
+
+
+
+## Example [Transformers](https://github.com/huggingface/transformers) DeepSparse Flask Deployment
+
+To illustrate how the DeepSparse Engine can be used with Hugging Face 
+Transformers, this directory contains a sample model server and client. 
+
+The server uses Flask to create an app with the DeepSparse Engine hosting a
+compiled Hugging Face Transformer model.
+The client can make requests into the server returning inference results for 
+given inputs.
+
+### Server
+
+First, start up the host `server.py` with your model of choice, SparseZoo stubs are
+also supported.
+
+Example command:
+```bash
+python server.py \
+    zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-moderate
+```
+
+You can leave that running as a detached process or in a spare terminal.
+
+This starts a Flask app with the DeepSparse Engine as the inference backend, accessible at `http://0.0.0.0:5543` by default.
+
+The app exposes HTTP endpoints at:
+- `/info` to get information about the compiled model
+- `/predict` to send inputs to the model and receive a response.
+    The number of inputs should match the compiled model's batch size.
+
+For a full list of options, run `python server.py -h`.
+
+Currently, the server uses DeepSparse-HuggingFace pipeline integration 
+for end-to-end prediction.  
+
+### Client
+
+`client.py` provides a callable `PipelineClient` object to make requests to the 
+server easy.
+The file is self-documented.  See example usage below:
+
+```python
+from client import PipelineClient
+remote_model = PipelineClient(address='0.0.0.0', port='5543')
+model_outputs = remote_model(question="What's my name?", context="My name is Snorlax")
+```
+
+### SparseZoo Stubs
+
+
+| Model Name     |      Stub      |
+|----------|-------------|
+| [bert-pruned-moderate]() | zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-moderate |
+| [bert-6layers-aggressive-pruned]()| zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_6layers-aggressive_96 |
+| [bert-pruned-conservative]()| zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-conservative |
+| [pruned_6layers-moderate]() | zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_6layers-moderate |
+| [pruned-aggressive_94]() | zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-aggressive_94|
+| [pruned_6layers-conservative]()| zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_6layers-conservative|
+| [bert-base]()|zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none |
```

Comment:
the links (ie `[bert-pruned-moderate]()`) don't go anywhere. we could remove them for now, but would also be nice to link out to their SparseZoo model cards.

Reply:
Done!